skyfield/jpllib.py

```diff
@@ -204,8 +206,21 @@ def __repr__(self):
 
 class ChebyshevPosition(SPICESegment):
     def _at(self, t):
-        position, velocity = self.spk_segment.compute_and_differentiate(t.tdb)
-        return position / AU_KM, velocity / AU_KM, None, None
+        try:
+            position, velocity = self.spk_segment.compute_and_differentiate(t.tdb)
+            return position / AU_KM, velocity / AU_KM, None, None
+        except OutOfRangeError as e:
+            start_time = t.ts.tdb(jd=self.spk_segment.start_jd)
+            end_time = t.ts.tdb(jd=self.spk_segment.end_jd)
+            out_of_range_times = []
+
+            for i, time in enumerate(t):
+                if e.out_of_range_times[i]:
+                    out_of_range_times.append(time)
+
+            raise OutOfRangeTimeError('out of range times: must be between %s and %s' % (start_time.utc_iso(),
```

Comment:
Perfect! The big question they'll have, "what times are supported by the ephemeris?", is answered right in the text of the exception. This will be an improvement for users!